eks_addon/release.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+# Builds, packages, and pushes artifacts for the Splunk OpenTelemetry Collector EKS Add-on.
+#
+# Parameters:
+#   --dry-run : (Optional) Enables dry-run mode that skips pushing artifacts.
+#   --chart-version VERSION : (Optional) Overrides the chart version from Chart.yaml.
+
+# Enable bash strict mode to fail fast
+set -euo pipefail
+
+# Check required tools
+for tool in "okta-aws-login" "aws" "yq" "helm" "docker"; do
+  command -v "${tool}" &>/dev/null || { echo "❌ Required command '${tool}' is not installed or not in PATH"; exit 1; }
+done
+
+# Parse command line arguments
+DRY_RUN_PREFIX=""
+CHART_VERSION=""
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --dry-run)
+      DRY_RUN_PREFIX="echo 🚧 [DRY-RUN] "
+      shift
+      ;;
+    --chart-version)
+      CHART_VERSION="$2"
+      echo "ℹ️ Chart version override provided: $CHART_VERSION"
+      shift 2
+      ;;
+    *)
+      echo "❌ Unknown argument: $1"
+      exit 1
+      ;;
+  esac
+done
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+CHART_DIR="$SCRIPT_DIR/../helm-charts/splunk-otel-collector"
+if [[ -z "$CHART_VERSION" ]]; then
+  CHART_VERSION=$(yq e ".version" "${CHART_DIR}/Chart.yaml")
+fi
+CHART_APPVERSION=$(yq e ".appVersion" "${CHART_DIR}/Chart.yaml")
+ECR_REGISTRY="709825985650.dkr.ecr.us-east-1.amazonaws.com"
+ECR_REGION="us-east-1"
```

Comment:
AWS_ACCOUNT_ID="${AWS_ACCOUNT_ID:-709825985650}"

```suggestion
ECR_REGION="${ECR_REGION:-us-east-1}"
ECR_REGISTRY="709825985650.dkr.ecr.${ECR_REGION}.amazonaws.com"
```
also, this account id is safe to have in the repo? is this the ecr url used by addon users?

General question - will aws make the image available in other regions?

Reply:
> General question - will aws make the image available in other regions?

I guess there is some CDN in place

Reply:
> also, this account id is safe to have in the repo? is this the ecr url used by addon users?

Yes, it's the common account for addon users

Reply:
AWS making it available in different regions would be ideal. I was worried we'd have to run this per-region. Thanks